### PR TITLE
Add airrflow docs and fix scrnaseq

### PR DESCRIPTION
**Pull request by ggabernet**

### CODE REVIEW

File: docs/markdown/pipelines/airrflow.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# BCRseq / TCRseq analysis
+
+To perform bulk amplicon BCRseq / TCRseq data analysis you can use the [nf-core/airrflow](https://nf-co.re/airrflow) pipeline.
+To perform single-cell BCRseq / TCRseq data analysis with data generated with the 10x genomics chromium (e.g. immune profiling libraries orr similar), you will have to run the [nf-core/scrnaseq](https://nf-co.re/scrnaseq) pipeline first running the `cellranger-multi` subworkflow. Afterwards, you can use the [nf-core/airrflow](nf-co.re/airrflow) pipeline departing from the AIRR format tables provided as output by `cellranger multi` to perform downstream analysis.
```
Comment (FriederikeHanssen) [inline docs/markdown/pipelines/airrflow.md:4]:
```suggestion
To perform single-cell BCRseq / TCRseq data analysis with data generated with the 10x genomics chromium (e.g. immune profiling libraries or similar), you will have to run the [nf-core/scrnaseq](https://nf-co.re/scrnaseq) pipeline first running the `cellranger-multi` subworkflow. Afterwards, you can use the [nf-core/airrflow](nf-co.re/airrflow) pipeline departing from the AIRR format tables provided as output by `cellranger multi` to perform downstream analysis.
```

**Review by FriederikeHanssen — APPROVED** [review]

File: docs/markdown/pipelines/airrflow.md (hunk 1 above)
Comment (louperelo) [inline docs/markdown/pipelines/airrflow.md:4]:
```suggestion
To perform single-cell BCRseq / TCRseq data analysis with data generated with the 10x genomics chromium (e.g. immune profiling libraries or similar), you will have to run the [nf-core/scrnaseq](https://nf-co.re/scrnaseq) pipeline first with the `cellranger-multi` subworkflow. Afterwards, you can use the [nf-core/airrflow](nf-co.re/airrflow) pipeline departing from the AIRR format tables provided as output by `cellranger multi` to perform downstream analysis.
```

File: docs/markdown/pipelines/scrnaseq.md [hunk 2]
```diff
@@ -9,4 +9,6 @@ For a comprehensive understanding of scRNAseq data processing, check out these c
 
 ## Pipeline
 
-Info about pipeline
+For single-cell RNAseq analysis please use the [nf-core/scrnaseq](nf-co.re/scrnaseq) pipeline. We currently recommend running the cellranger tool as it's the most requested one and currently compatible with our downstream analysis scripts.
+
+For secondary analysis, we are currently just running custom R and Rmarkdown files using the Seurat library. Please ask for example scRNAseq projects to your team leader.
```
Comment (louperelo) [inline docs/markdown/pipelines/scrnaseq.md:14]:
```suggestion
For secondary analysis, we are currently just running custom R and Rmarkdown files using the Seurat library. Please ask your team leader for example scRNAseq projects.
```

File: docs/index.rst [hunk 3]
```diff
@@ -36,7 +36,10 @@ Analysis types
 :doc:`markdown/pipelines/scrnaseq`
    Single-cell RNA sequencing
 
-:doc: `markdown/pipelines/guideseq`
+:doc:`markdown/pipelines/airrflow`
+   BCR / TCR sequencing
+
+:doc:`markdown/pipelines/guideseq`
```
Comment (louperelo) [inline docs/index.rst:42]: Not sure where this part will appear on the page, but should `Sarek` be added in this list as well? There was another PR today adding information on how to use the pipeline.

**Review by louperelo — APPROVED** [review]

File: docs/index.rst (hunk 3 above)
Comment (ggabernet) [reply docs/index.rst:42]: It's already there in the WES / WGS sequencing page

**Pull request merged**